specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2018-04-04/policyEvents.json

```diff
@@ -30,7 +30,7 @@
     }
   },
   "paths": {
-    "/{managementGroupId}/providers/Microsoft.PolicyInsights/policyEvents/{policyEventsResource}/queryResults": {
+    "/providers/{managementGroupsNamespace}/managementGroups/{managementGroupName}/providers/Microsoft.PolicyInsights/policyEvents/{policyEventsResource}/queryResults": {
```

Comment:
autorest/oav validation does not like it. Errors out saying that a spec can only be defined for one namespace. This is the workaround I had found in our previous release. It results in identical API; since it is a single valued enum, it turns into a const string part of path.

Reply:
As a note, there is no change here from previous version. The change you pointed out the comment on was done on the previous version to fix a different validation issue where multiple paths with matching template but different path parameter names (e.g. managementGroupId, vs. policyDefinitionId, etc.) was not acceptable.